### PR TITLE
Styling the search/info/card components

### PR DESCRIPTION
**Pull request by tawfiknasser**
in this PR : 
- we replaced the px with percents % so it can fit different sizes of screens,
- we added flex container and flex box for the containers

### CODE REVIEW

File: client/src/components/Header_Bar/header.css [hunk 1]
```diff
@@ -1,18 +1,19 @@
 .HeadBar {
   position: absolute;
-  width: 375px;
+  width: 100%;
   height: 57px;
   left: 0px;
   top: 0px;
```
Comment (shiryz) [inline client/src/components/Header_Bar/header.css]: Please lint :fire:

Comment (tawfiknasser) [reply client/src/components/Header_Bar/header.css]: we tested our product on Samsung 6 edge and iphone 8 ,
and it's working , i don't say perfectly . we need to refactor later it's MVP for now.
and only for MOBILE .
we will try to use only percents in the future as you mentioned , to make it fit also for tablets.

**Comment by shiryz** [comment]
The styling for this one is not centered 
screenshot from 2018-09-13 16-29-40

**Comment by shiryz** [comment]
Happy that it's centered now, but it doesn't match  your designs anymore
Is this the final design you're going for?

**Comment by tawfiknasser** [comment]
updated the design similar to ~ figma,
we will redesign it later also , with figma . different colors different margins.

**Pull request merged**